main.js

```diff
@@ -0,0 +1,56 @@
+document.getElementById('submit').addEventListener('click', function () {
+  var name = document.getElementById('name').value;
+  var text = document.getElementById('message').value;

```

Comment:
Be as descriptive as you can with your variables so others know exactly what is in them